packages/browser/src/backend.ts

```diff
@@ -76,6 +77,7 @@ export class BrowserBackend extends BaseBackend<BrowserOptions> {
       // which is much better than creating new group when any key/value change
       const ex = exception as {};
       event = eventFromPlainObject(ex, hint.syntheticException);
+      addExceptionTypeValue(event, `${ex}`);
```

Comment:
This will always say `[object Object]` which isn't the best I guess. Maybe we can put custom string here?